content/actions/using-workflows/workflow-syntax-for-github-actions.md

```diff
@@ -979,12 +979,15 @@ services:
 
 The Docker image to use as the service container to run the action. The value can be the Docker Hub image name or a registry name.
 
+> [!NOTE]
```

Comment:
Is this a new pattern within this document? Wondering whether we need to duplicate this information here, since it's already [listed here](https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability)

Reply:
Wow, did not see this section before. I can see it was committed around a month ago so this is probably why. You are right. I'll drop this.

Reply:
Actually if allowed context are specified clearly already. I will close this PR.